frontend/src/components/form/internal/InputMaskInternal.vue

```diff
@@ -14,6 +15,15 @@ const { label, name, mask, placeholder, disabled, bold } = defineProps<{
 }>();
 
 const { errorMessage, value } = useField<string>(name);
+
+const normalizedValue = computed({
+  get: () => {
+    return value.value === '' ? undefined : value.value;
+  },
+  set: (val) => {
+    value.value = val === '' || val === undefined ? (undefined as unknown as string) : val;
+  }
+});
```

Comment:
Have you checked to see if this might break form submissions in other components? This is a widespread change to every view that uses an input field.

Reply:
It's just for the InputMask field not any other input fields, and it's only changing the it so that it handles no input as undefined rather than it's default of empty string, which triggers autosave, as empty string is technically a value. I tested other InputMasks and all seems fine, and test seem to be passing at same levels as before.